packages/otel/src/index.ts

```diff
@@ -8,7 +8,7 @@ import {
   ATTR_URL_FULL,
   ATTR_HTTP_ROUTE,
 } from '@opentelemetry/semantic-conventions'
-import type { Env, Input } from 'hono'
+import type { Env, HonoRequest, Input } from 'hono'
```

Comment:
This is change is not necessary. `HonoRequest` is not used.

Reply:
Oops 🙃